spec/integ/bridge.spec.js

```diff
@@ -187,6 +187,22 @@ describe("Bridge", function() {
             });
         });
 
+        it("should not create a room if roomId is returned from the function but should still store it",
+        async function(done) {
```

Comment:
I don't think you need `done` here. Some tests use it because they were written long before `async` was a thing.